### PR TITLE
when no package exists url params is wrong and shows 404 error page in docs - issue fix

### PR DESCRIPTION
**Pull request by Sreejith-Websphere**
#### Were the changes tested on
- [ ] Firefox (Desktop)
- [ ] Safari (Desktop)
- [ ] Chrome (Desktop)
- [ ] Internet Explorer (Desktop)
- [ ] iOS (Mobile)
- [ ] Android (Mobile)
#### Running validation tools
- [ ] https://validator.w3.org/checklink
- [ ] https://validator.w3.org
- [ ] IBM Equal Access Accessibility Checker (https://www.ibm.com/able/toolkit/verify/automated)
- [ ] Lighthouse (in Chrome dev tools)

### CODE REVIEW

**Comment by Sreejith-Websphere** [comment]
Found issue in draft site - when there is no package the url query params goes like this ?package=/package-frame.html&class=overview-tree.html the shows 404 page Refer below screenshot. Below issue is fixed. For testing  url - https://draft-openlibertyio.mybluemix.net/docs/modules/reference/liberty-jakartaee9.1-javadoc or https://draft-openlibertyio.mybluemix.net/docs/modules/reference/liberty-jakartaee9.1-javadoc/overview-tree.html
Screenshot 2022-05-05 at 3 05 27 PM

**Comment by Sreejith-Websphere** [comment]
Screenshot 2022-05-06 at 1 20 03 AM

tested in demo1

**Review by kinueng — COMMENTED** [review]
The `javadoc-redirect.js` file is starting to have too much duplicated logic. Instead of adding three almost identical if-check-else, can we refactor this file to be more modular? 

A suggestion is to start small and refactor the new `else` statement into a method shared between three use-cases: `liberty-javaee`, `liberty-jakartaee`, and `microprofile`

**Comment by Sreejith-Websphere** [comment]
@kin code refactored

File: src/main/content/_assets/js/javadoc-redirect.js [hunk 1]
```diff
@@ -65,4 +76,19 @@ $(function () {
...
+function ifPackageNotExists(port,api,version,jd) {
+  window.top.location.href =
+  'https://' +
+  window.top.location.hostname +
+  port +
+  '/docs/latest/reference/javadoc/'+ api +
+  version +
+  '-javadoc.html?class=' +
+  jd;
+}
```
Comment (kinueng) [inline src/main/content/_assets/js/javadoc-redirect.js]: Nit-pick: This file needs an end of line

Comment (Sreejith-Websphere) [reply src/main/content/_assets/js/javadoc-redirect.js]: added

**Review by kinueng — APPROVED** [review]
LGTM

**Pull request merged**